sonic_platform_base/chassis_base.py

```diff
@@ -54,9 +57,25 @@ def __init__(self):
         # List of ThermalBase-derived objects representing all thermals
         # available on the chassis
         self._thermal_list = []
+
         self._voltage_sensor_list = []
         self._current_sensor_list = []
 
+        # Initialize voltage and current sensors lists from data file if available
```

Comment:
ok..can be done.. however that will need change in daemon repo. Will check and send another PR.

Reply:
@bmridul here is one way

<img width="306" alt="image" src="https://github.com/sonic-net/sonic-platform-common/assets/45705344/7618b0a5-32d8-4415-98b9-30b0d7f7293b">

Reply:
@bmridul daemon repo change is fine

Reply:
@prgeor I looked into this. Sensormond retrieves the sensor list from chassis and module classes which are implemented by platform. If we take this change in Sensormond code, it will have to overwrite the list provided by the platform and modify the chassis implementation. This, though technically doable in python, is not clean object oriented design. I think its better here as this is directly provided in the base chassis class.

Place where the sensor list would can get overwritten into chassis class..

https://github.com/sonic-net/sonic-platform-daemons/blob/d8977f3608e5c9263d9b9d9b33087890637ac436/sonic-sensormond/scripts/sensormond#L457

Code where it is referenced.

https://github.com/sonic-net/sonic-platform-daemons/blob/d8977f3608e5c9263d9b9d9b33087890637ac436/sonic-sensormond/scripts/sensormond#L196

Let me know what u think.

Reply:
@bmridul lets discuss. sensormond is the only one standing odd for its implementation details in chassis_base.py which is completely abstract.

Reply:
I have put this login in Sensormond. Sensormond looks at this in addition to the chassis class, so the chassis API does not need to change. Will send sensormond PR shortly.